internal/gps/identifier.go

```diff
@@ -148,6 +148,10 @@ func (i ProjectIdentifier) errString() string {
 	return fmt.Sprintf("%s (from %s)", i.ProjectRoot, i.Source)
 }
 
+func (i ProjectIdentifier) String() string {
+	return i.errString()
```

Comment:
Any objections to this `errString` format becoming the canonical exported String method format? `fmt.Sprintf("%s (from %s)", i.ProjectRoot, i.Source)`

`errString()` has 36 usages, all of which are passed to formatting functions with `%s`, so I'd like to do a follow up PR to just absorb that method into this one and let them all implicitly call this.

Reply:
mmm yes, i like this. the individual properties are accessible already - callers can easily construct their own output if they so choose. 👍

Reply:
(separate PR, though)